engine/app/components/citizens_advice_components/sidebar.html.haml

```diff
@@ -0,0 +1,14 @@
+%aside.cads-sidebar
+  .cads-sidebar__content
+    %p.cads-sidebar__title= title
+    - if section_title.url.present?
```

Comment:
I think this section (line 4 to 9) needs to be changed a bit. It also threw me an error (which is why the test failed) but I couldn't figure out how to re-write it using &. (especially the `else` statement). I also saw that some of the other haml files also use if statements so not sure how to fix this one

`Run bundle exec haml-lint
app/components/citizens_advice_components/sidebar.html.haml:10 [W] RuboCop: Style/SafeNavigation: Use safe navigation (&.) instead of checking if an object exists before calling the method.`